chain/store/store.go

```diff
@@ -446,6 +449,43 @@ func (cs *ChainStore) takeHeaviestTipSet(ctx context.Context, ts *types.TipSet)
 	return nil
 }
 
+// FlushValidationCache removes all results of block validation from the
+// chain metadata store. Usually the first step after a new chain import.
+func (cs *ChainStore) FlushValidationCache() error {
+	log.Infof("clearing block validation cache...")
+
+	dsWalk, err := cs.ds.Query(query.Query{KeysOnly: true})
```

Comment:
```suggestion
	dsWalk, err := cs.ds.Query(query.Query{KeysOnly: true, Prefix: blockValidationCacheKeyPrefix.String()})
```

Then we should be able to drop the prefix check below

Reply:
Prefix won't work :( 
Prefix is for `foo/bar`, it doesn't work for `foo:bar` which is how we assemble the validation cache via `.Instance` higher up

Reply:
@magik6k added a comment to clarify why things are the way they are: https://github.com/filecoin-project/lotus/pull/4287/commits/23f5a99117226fba70fea235c421f1d7b6f26745